### PR TITLE
Support for transient frame updates

### PR DESCRIPTION
**Pull request by jonatanklosko**
https://user-images.githubusercontent.com/17034772/226475017-301135f0-19eb-4df5-8d24-29c1809c9567.mov

### CODE REVIEW

File: lib/kino/frame.ex [hunk 1]
```diff
@@ -64,11 +64,34 @@ defmodule Kino.Frame do
       option is useful when updating frame in response to client
       events, such as form submission
 
+    * `:history` - if the update will be part of the frame history.
```
Comment (jonatanklosko) [inline lib/kino/frame.ex:67]: I'm leaning towards `:transient`, wdyt?

Comment (aleDsz) [reply lib/kino/frame.ex:67]: +1 for `:transient`

Comment (josevalim) [reply lib/kino/frame.ex:67]: I like `:history` for simplicity. If folks dislike it, `:temporary` perhaps? `:transient` is too fancy. :D

**Review by josevalim — APPROVED** [review]

File: lib/kino/frame.ex (hunk 1 above)
Comment (jonatanklosko) [reply lib/kino/frame.ex:67]: I just think `:history` doesn't read as well, especially with `clear`. `:temporary` also works for me :)

**Review by aleDsz — APPROVED** [review]

**Pull request merged**